### DISCUSSION TITLE
Can't open files/folders with names containing parantheses (curved brackets)

### DISCUSSION THREAD

**Discussion by bierbrezen** [filebrowser/filebrowser#1334] [Q&A] [answered]
I noticed that I can't open files and folders which names contains parantheses, "(" ")", e.g. some of my music files and album folders.
If I try to open them, it only shows "This location can't be reached."
 
Would it be possible to change that in a future version?

**Comment by Frank4411** [comment]
I was not able to recreate this error on version 2.12.0. What version are you on?

I tested with both an admin account and a non admin account and saw no issue on either.

Some folder names I tried:
test (2019)}}{}]]][][][))))9
test (2019)
test {2019}
test {2019}

**Comment by bierbrezen** [comment]
So i guess this is not a standard behaviour.
I use latest version 2.13.0. I installed filebrowser as container on portainer on a RPI4 on raspbian buster 10. I also use an admin account (single user).

Maybe it has something to do that I reverse proxied filebrowser to a subfolder. https://xx.xx.xx.xx/filebrowser/.
But everything works fine with opening and creating files and folders without parantheses - it even works with square and curly brackets but not with round ones.

---
I also tried with accessing on standard port now - without reverse proxy. Unfortunately with same issue.

**Reply by Frank4411** [reply]
I have had some issues in the past doing reverse proxies with a sub folder on other self hosted projects. Have you tried to access the service directly. Like bypassing the reverse proxy, does that seem to work correctly?

**Reply by bierbrezen** [reply]
I tried out something und found out that with baseURL configuration "" on .filebrowser.json I didn't have that issue either,

but with configuration needed for reverse proxying to subfolder "/filebrowser" the described issue occurs, even when accessing directly on local address/port.

My .filebrowser.json:

{
"port": 8880,
"baseURL": "/filebrowser",
"address": "",
"log": "stdout",
"database": "/database.db",
"root": "/srv"
}

**Comment by bierbrezen** [comment]
Today I tried it again. I set baseURL to "" in .json file and declarated the subfolder as environment variable in docker container ("--baseurl /filebrowser") to get it out of the config file. Filebrowser still works but I got the same issue when I was trying to open ("This location can't be reached.") or create ("404 page not found") a file or folder with a name containing round brackets.

**Comment by M4he** [comment, marked as answer]
Hey @bierbrezen I'm facing the same issue with reverse proxy configurations. Seems to be an issue for versions later than v2.11.0 currently, see here: https://github.com/filebrowser/filebrowser/issues/1332#issuecomment-801439539

**Reply by M4he** [reply]
v2.14 has been released and fixes this.

**Reply by bierbrezen** [reply]
Confirmed! Updated to 2.14. Issue is fixed.

Thank you!

**Comment by bierbrezen** [comment]
@M4he:
Rolling back to version v2.11.0-pi solved it for me!

I'd really appreciate it if this could be fixed again in later versions.

Thank you very much!
